README.md

```diff
@@ -10,7 +10,8 @@ It is primarily focused on FTA which offers possibilities for system partonomy d
 ```shell script
 npm install
 npm run build
-npm run preview
+npm run preview (only preview)
+npm run dev (for development)
```

Comment:
We do not need these comments

Reply:
When i was trying to start project first for time, i did not find correct command for it

Reply:
Sorry, I meant the commands are correct, just not what's between the parenthesis

Reply:
Also, if we add `npm run dev`, we should specify that we should use either `npm run install && npm run dev` **or** `npm run install && npm run build && npm run preview`

Reply:
How do u start the project ? I was using npm install && npm run dev. Is it correct ?

Reply:
Also here is one failing check, not sure how can i fix it

Reply:
I hopefully fixed failing checks so when you commit next it should be fixed.

Reply:
@Kasmadei I addressed your need to have some development notes in https://github.com/kbss-cvut/fta-fmea-ui/pull/191

Reply:
resolved in #191